packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import RatingItem from './RatingItem'
+
+export default class RatingBar extends Component {
+  static propTypes = {
+    theme: PropTypes.object,
+    ratingScale: PropTypes.number,
+    rating: PropTypes.number,
+    backgroundColor: PropTypes.string,
+    total: PropTypes.string,
+    stroke: PropTypes.string,
+    strokeWidth: PropTypes.number,
+    itemHeight: PropTypes.string,
+    itemWidth: PropTypes.string,
+    pathD: PropTypes.string.isRequired,
+    viewBox: PropTypes.string.isRequired,
```

Comment:
I'm not sure its a good idea to require svg specific stuff like `pathD` and `viewBox`.  Ideally these concepts should be abstracted away.  In my opinion this component is currently too complicated for general use.